authorize_helper.go

```diff
@@ -180,7 +180,7 @@ func IsRedirectURISecure(ctx context.Context, redirectURI *url.URL) bool {
 // IsRedirectURISecureStrict is stricter than IsRedirectURISecure and it does not allow custom-scheme
 // URLs because they can be hijacked for native apps. Use claimed HTTPS redirects instead.
 // See discussion in https://github.com/ory/fosite/pull/489.
-func IsRedirectURISecureStrict(redirectURI *url.URL) bool {
+func IsRedirectURISecureStrict(ctx context.Context, redirectURI *url.URL) bool {
```

Comment:
Because you added it to `IsRedirectURISecure`. :-) So `IsRedirectURISecure` is used for `RedirectSecureChecker` config and `IsRedirectURISecureStrict` is a stricter version of `IsRedirectURISecure`. So if one wants to use it as a config value, currently there is a signature mismatch.

So this is just updating the signature to be compatible with `RedirectSecureChecker`.

Reply:
I see! That makes sense